cmd/sovereignnode/sovereignNodeRunner.go

```diff
@@ -1670,6 +1671,7 @@ func createWhiteListerVerifiedTxs(generalConfig *config.Config) (process.WhiteLi
 func createSovereignWsReceiver(
 	dataPool dataRetriever.PoolsHolder,
 	config *sovereignConfig.NotifierConfig,
+	startRound uint64,
```

Comment:
mainChainStartRound?

Reply:
Right, renamed it to MainChainNotarizationStartRound